### PR TITLE
Re-add global interface messages for default user groups

### PR DESCRIPTION
**Pull request by dmehus**
With thanks to @AgentIsai for https://meta.miraheze.org/wiki/User_groups, building upon what I started at https://revi.wiki/wiki/%EC%82%AC%EC%9A%A9%EC%9E%90:Dmehus/ManageWiki/Default_Permissions

### CODE REVIEW

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
For groups that have default messages, these need added to https://github.com/miraheze/MirahezeMagic/blob/master/i18n/overrides/en.json and the qqq.json equivalent. (Prefixed with `miraheze-`) Then each message needs listed in: https://github.com/miraheze/MirahezeMagic/blob/d96e751c67d9fdc100d6d57e55cb8729d7aedea2/includes/MirahezeMagicHooks.php#L167-L183

This is the *only* way to do this, however doing so will prevent local overrides by editing the normal interface messages, only editing the `miraheze-` prefixed messages will work for overrides therefore I don't personally like the idea to much, but that's how to do it.

**Comment by dmehus** [comment]
> For groups that have default messages, these need added to https://github.com/miraheze/MirahezeMagic/blob/master/i18n/overrides/en.json and the qqq.json equivalent. (Prefixed with `miraheze-`) Then each message needs listed in:
> 
> https://github.com/miraheze/MirahezeMagic/blob/d96e751c67d9fdc100d6d57e55cb8729d7aedea2/includes/MirahezeMagicHooks.php#L167-L183
> 
> This is the _only_ way to do this, however doing so will prevent local overrides by editing the normal interface messages, only editing the `miraheze-` prefixed messages will work for overrides therefore I don't personally like the idea to much, but that's how to do it.

I'm actually not sure we _need_ to do this, as I can say with 100% certainty we've previously used regular global interface messages for the grouppages for the default user groups, but they pointed to the Meta: namespace pages, rather than the default User groups page Agent Isai made

**Comment by Universal-Omega** [comment]
I looked at history going back 2 years, as far as I can say we have never done this for groups that have defaults.

File: i18n/overrides/en.json [hunk 1]
```diff
@@ -20,5 +20,20 @@
...
+	"miraheze-group-bureaucrat-member": "{{GENDER:$1|bureaucrat}}",
+	"miraheze-group-sysop": "Administrators",
+    	"miraheze-grouppage-sysop": "m:Special:MyLanguage/User groups#Administrators",
+	"miraheze-group-sysop-member": "{{GENDER:$1|administrator}}",
+	"miraheze-group-interface-admin": "Interface administrators",
+    	"miraheze-grouppage-interface-admin": "m:Special:MyLanguage/User groups#Interface administrators",
+	"miraheze-group-interface-admin-member": "{{GENDER:$1|interface administrator}}",
+	"miraheze-group-bot": "Bots",
+    	"miraheze-grouppage-bot": "m:Special:MyLanguage/User groups#Bots",
+	"miraheze-group-bot-member": "{{GENDER:$1|bot}}",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:38]: Remove the "," as that is a syntax error.

File: i18n/overrides/qqq.json [hunk 2]
```diff
@@ -13,5 +13,20 @@
...
+	"miraheze-group-bureaucrat-member": "{{GENDER:$1|bureaucrat}}",
+	"miraheze-group-sysop": "Administrators",
+    	"miraheze-grouppage-sysop": "m:Special:MyLanguage/User groups#Administrators",
+	"miraheze-group-sysop-member": "{{GENDER:$1|administrator}}",
+	"miraheze-group-interface-admin": "Interface administrators",
+    	"miraheze-grouppage-interface-admin": "m:Special:MyLanguage/User groups#Interface administrators",
+	"miraheze-group-interface-admin-member": "{{GENDER:$1|interface administrator}}",
+	"miraheze-group-bot": "Bots",
+    	"miraheze-grouppage-bot": "m:Special:MyLanguage/User groups#Bots",
+	"miraheze-group-bot-member": "{{GENDER:$1|bot}}",
```
Comment (Universal-Omega) [inline i18n/overrides/qqq.json:31]: Remove the "," as that is a syntax error.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/miraheze/en.json [hunk 3]
```diff
@@ -92,52 +92,52 @@
...
-	"grouppage-trustandsafety": "m:Trust and Safety",
+	"grouppage-trustandsafety": "m:Special:MyLanguage/Trust and Safety",
 	"group-global-ipblock-exempt": "Global IP block exemption",
 	"group-global-ipblock-exempt-member": "{{GENDER:$1|global IP block exempt}}",
-	"grouppage-global-ipblock-exempt": "m:Global IP block exemption",
+	"grouppage-global-ipblock-exempt": "m:Special:MyLanguage/Global IP block exemption",
 	"group-member": "Members",
  	"group-member-member": "{{GENDER:$1|member}}",
+	"grouppage-member": "m:Special:MyLanguage/User groups#Members",
+	"grouppage-user": "m:Special:MyLanguage/User groups#Users",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:140]: grouppage-user should be in overrides I believe.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/miraheze/qqq.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
 	"miraheze-helpcenter": "Link for the help center page.",
 	"miraheze-requestfeatures": "Request features on a wiki",
 	"miraheze-requestwiki": "Request a wiki",
-	"miraheze-sidebar-donate": "Sidebar header for donate.\n{{Identical|Donate}}",
```
Comment (Universal-Omega) [inline i18n/miraheze/qqq.json:27]: You removed `-donate` probably by accident.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: includes/MirahezeMagicHooks.php [hunk 5]
```diff
@@ -180,6 +180,22 @@ public static function onMessageCacheGet( &$lcKey ) {
 			'centralauth-login-error-locked',
 			'snapwikiskin',
 			'skinname-snapwikiskin',
+			"miraheze-group-checkuser": "Checkusers",
```
Comment (Universal-Omega) [inline includes/MirahezeMagicHooks.php:183]: The messages here should not be prefixed by `miraheze-` and you aren't adding values. Plus this isn't JSON anymore.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: includes/MirahezeMagicHooks.php (hunk 5 above)
Comment (dmehus) [reply includes/MirahezeMagicHooks.php:183]: Oh, right. Okay, that makes sense. Thanks. :)

File: i18n/miraheze/qqq.json [hunk 6]
```diff
@@ -37,38 +37,39 @@
...
 	"group-wikicreator-member": "{{GENDER:$1|wiki creator}}",
-	"grouppage-wikicreator": "m:Wiki creators",
+	"grouppage-wikicreator": "m:Special:MyLanguage/Meta:Wiki creators",
 	"group-wikimanager": "Wiki Managers",
 	"group-wikimanager-member": "{{GENDER:$1|wiki manager}}",
-	"grouppage-wikimanager": "m:Wiki managers",
+	"grouppage-wikimanager": "m:Special:MyLanguage/Wiki managers",
 	"group-rollbacker": "Rollbackers",
-	"grouppage-rollbacker": "m:Rollbacker",
+    	"grouppage-rollbacker": "m:Special:MyLanguage/User groups#Rollbackers",
```
Comment (Universal-Omega) [inline i18n/miraheze/qqq.json:54]: Tabs not spaces

File: i18n/miraheze/qqq.json [hunk 7]
```diff
@@ -37,38 +37,39 @@
...
 	"group-rollbacker-member": "{{GENDER:$1|rollbacker}}",
-	"group-autopatrolled": "Autopatrollers",
-	"grouppage-autopatrolled": "m:Meta:Autopatrolled",
+	"group-autopatrolled": "Autopatrolled users",
+    	"grouppage-autopatrolled": "m:Special:MyLanguage/User groups#Autopatrolled",
```
Comment (Universal-Omega) [inline i18n/miraheze/qqq.json:57]: Tabs not spaces

File: i18n/overrides/en.json [hunk 8]
```diff
@@ -20,5 +20,21 @@
 	"miraheze-restriction-delete": "Delete",
 	"miraheze-skinname-snapwikiskin": "Snap!",
 	"miraheze-snapwikiskin": "Snap!",
-	"miraheze-wikibase-sitelinks-miraheze": "Miraheze Wikis"
+	"miraheze-wikibase-sitelinks-miraheze": "Miraheze Wikis",
+	"miraheze-group-checkuser": "Checkusers",
+	"miraheze-group-checkuser-member": "checkuser",
+	"miraheze-grouppage-checkuser": "m:Special:MyLanguage/CheckUser",
+	"miraheze-group-bureaucrat": "Bureaucrats",
+    	"miraheze-grouppage-bureaucrat": "m:Special:MyLanguage/User groups#Bureaucrats",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:28]: Tabs not spaces

File: i18n/overrides/en.json [hunk 9]
```diff
@@ -20,5 +20,21 @@
...
+	"miraheze-group-bureaucrat-member": "{{GENDER:$1|bureaucrat}}",
+	"miraheze-group-sysop": "Administrators",
+    	"miraheze-grouppage-sysop": "m:Special:MyLanguage/User groups#Administrators",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:31]: Tabs not spaces

File: i18n/overrides/en.json [hunk 10]
```diff
@@ -20,5 +20,21 @@
...
+	"miraheze-group-sysop-member": "{{GENDER:$1|administrator}}",
+	"miraheze-group-interface-admin": "Interface administrators",
+    	"miraheze-grouppage-interface-admin": "m:Special:MyLanguage/User groups#Interface administrators",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:34]: Tabs not spaces

File: i18n/overrides/en.json [hunk 11]
```diff
@@ -20,5 +20,21 @@
...
+	"miraheze-group-interface-admin-member": "{{GENDER:$1|interface administrator}}",
+	"miraheze-group-bot": "Bots",
+    	"miraheze-grouppage-bot": "m:Special:MyLanguage/User groups#Bots",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:37]: Tabs not spaces

File: i18n/overrides/qqq.json [hunk 12]
```diff
@@ -13,5 +13,21 @@
 	"miraheze-shoutwiki-loginform-tos": "I have read, understood and agree to the [https://meta.miraheze.org/wiki/Terms_of_Use Terms of Use] and [https://meta.miraheze.org/wiki/Privacy_Policy Privacy Policy]",
 	"miraheze-shoutwiki-must-accept-tos": "You must accept the Terms of Use in order to be able to create an account!",
 	"miraheze-skinname-snapwikiskin": "The name of a graphical user interface skin. An [[:w:Onomatopoeia|onomatopoeia]]. Translation optional.",
-	"miraheze-snapwikiskin": "The name of a graphical user interface skin. An [[:w:Onomatopoeia|onomatopoeia]]. Translation optional."
+	"miraheze-snapwikiskin": "The name of a graphical user interface skin. An [[:w:Onomatopoeia|onomatopoeia]]. Translation optional.",
+	"miraheze-group-checkuser": "Checkusers",
+	"miraheze-group-checkuser-member": "checkuser",
+	"miraheze-grouppage-checkuser": "m:Special:MyLanguage/CheckUser",
+	"miraheze-group-bureaucrat": "Bureaucrats",
+    	"miraheze-grouppage-bureaucrat": "m:Special:MyLanguage/User groups#Bureaucrats",
```
Comment (Universal-Omega) [inline i18n/overrides/qqq.json:21]: Tabs not spaces

File: i18n/overrides/qqq.json [hunk 13]
```diff
@@ -13,5 +13,21 @@
...
+	"miraheze-group-bureaucrat-member": "{{GENDER:$1|bureaucrat}}",
+	"miraheze-group-sysop": "Administrators",
+    	"miraheze-grouppage-sysop": "m:Special:MyLanguage/User groups#Administrators",
```
Comment (Universal-Omega) [inline i18n/overrides/qqq.json:24]: Tabs not spaces

File: i18n/overrides/qqq.json [hunk 14]
```diff
@@ -13,5 +13,21 @@
...
+	"miraheze-group-sysop-member": "{{GENDER:$1|administrator}}",
+	"miraheze-group-interface-admin": "Interface administrators",
+    	"miraheze-grouppage-interface-admin": "m:Special:MyLanguage/User groups#Interface administrators",
```
Comment (Universal-Omega) [inline i18n/overrides/qqq.json:27]: Tabs not spaces

File: i18n/overrides/qqq.json [hunk 15]
```diff
@@ -13,5 +13,21 @@
...
+	"miraheze-group-interface-admin-member": "{{GENDER:$1|interface administrator}}",
+	"miraheze-group-bot": "Bots",
+    	"miraheze-grouppage-bot": "m:Special:MyLanguage/User groups#Bots",
```
Comment (Universal-Omega) [inline i18n/overrides/qqq.json:30]: Tabs not spaces

File: includes/MirahezeMagicHooks.php [hunk 16]
```diff
@@ -180,6 +180,22 @@ public static function onMessageCacheGet( &$lcKey ) {
 			'centralauth-login-error-locked',
 			'snapwikiskin',
 			'skinname-snapwikiskin',
+			"group-checkuser": "Checkusers",
+			"group-checkuser-member": "checkuser",
+			"grouppage-checkuser": "m:Special:MyLanguage/CheckUser",
+			"group-bureaucrat": "Bureaucrats",
+    			"grouppage-bureaucrat": "m:Special:MyLanguage/User groups#Bureaucrats",
```
Comment (Universal-Omega) [inline includes/MirahezeMagicHooks.php:187]: Tabs not spaces

File: includes/MirahezeMagicHooks.php [hunk 17]
```diff
@@ -180,6 +180,22 @@ public static function onMessageCacheGet( &$lcKey ) {
...
+			"group-bureaucrat-member": "{{GENDER:$1|bureaucrat}}",
+			"group-sysop": "Administrators",
+    			"grouppage-sysop": "m:Special:MyLanguage/User groups#Administrators",
```
Comment (Universal-Omega) [inline includes/MirahezeMagicHooks.php:190]: Tabs not spaces

File: includes/MirahezeMagicHooks.php [hunk 18]
```diff
@@ -180,6 +180,22 @@ public static function onMessageCacheGet( &$lcKey ) {
...
+			"group-sysop-member": "{{GENDER:$1|administrator}}",
+			"group-interface-admin": "Interface administrators",
+    			"grouppage-interface-admin": "m:Special:MyLanguage/User groups#Interface administrators",
```
Comment (Universal-Omega) [inline includes/MirahezeMagicHooks.php:193]: Tabs not spaces

File: includes/MirahezeMagicHooks.php [hunk 19]
```diff
@@ -180,6 +180,22 @@ public static function onMessageCacheGet( &$lcKey ) {
...
+			"group-interface-admin-member": "{{GENDER:$1|interface administrator}}",
+			"group-bot": "Bots",
+    			"grouppage-bot": "m:Special:MyLanguage/User groups#Bots",
```
Comment (Universal-Omega) [inline includes/MirahezeMagicHooks.php:196]: Tabs not spaces

File: i18n/miraheze/qqq.json [hunk 20]
```diff
@@ -37,38 +37,39 @@
...
 	"group-confirmed-member": "{{GENDER:$1|confirmed user}}",
+   	"grouppage-confirmed": "m:Special:MyLanguage/User groups#(Auto)confirmed",
 	"group-proxybot": "Proxy Bot",
 	"group-proxybot-member": "proxy bot",
 	"grouppage-proxybot": "Meta:Proxybot",
 	"group-translator": "Translators",
 	"group-translator-member": "translator",
 	"group-translationadmin": "Translation administrators",
 	"group-translationadmin-member": "translation administrator",
+    	"grouppage-translationadmin": "m:Special:MyLanguage/Meta:Translation administrators",
```
Comment (Universal-Omega) [inline i18n/miraheze/qqq.json:69]: Tabs not spaces

File: i18n/miraheze/qqq.json [hunk 21]
```diff
@@ -37,38 +37,39 @@
...
+    	"grouppage-rollbacker": "m:Special:MyLanguage/User groups#Rollbackers",
 	"group-rollbacker-member": "{{GENDER:$1|rollbacker}}",
-	"group-autopatrolled": "Autopatrollers",
-	"grouppage-autopatrolled": "m:Meta:Autopatrolled",
+	"group-autopatrolled": "Autopatrolled users",
+    	"grouppage-autopatrolled": "m:Special:MyLanguage/User groups#Autopatrolled",
 	"group-autopatrolled-member": "{{GENDER:$1|autopatrolled}}",
 	"group-confirmed": "Confirmed users",
 	"group-confirmed-member": "{{GENDER:$1|confirmed user}}",
+   	"grouppage-confirmed": "m:Special:MyLanguage/User groups#(Auto)confirmed",
```
Comment (Universal-Omega) [inline i18n/miraheze/qqq.json:61]: Tabs not spaces

File: i18n/miraheze/en.json [hunk 22]
```diff
@@ -92,52 +92,51 @@
...
-	"grouppage-globalsysop": "m:Global Sysops",
+	"grouppage-globalsysop": "m:Special:MyLanguage/Global Sysops",
 	"group-proxybot": "Proxy Bot",
 	"group-proxybot-member": "proxy bot",
 	"grouppage-proxybot": "m:User:Proxybot",
 	"group-translator": "Translators",
 	"group-translator-member": "translator",
 	"group-translationadmin": "Translation administrators",
-        "grouppage-translationadmin": "Project:Translation administrators",
+        "grouppage-translationadmin": "m:Special:MyLanguage/Meta:Translation administrators",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:129]: Tabs not spaces

File: i18n/miraheze/en.json [hunk 23]
```diff
@@ -92,52 +92,51 @@
...
-        "grouppage-rollbacker": "Project:Rollbackers",
+        "grouppage-rollbacker": "m:Special:MyLanguage/User groups#Rollbackers",
 	"group-rollbacker-member": "{{GENDER:$1|rollbacker}}",
 	"group-autopatrolled": "Autopatrolled users",
-        "grouppage-autopatrolled": "Project:Autopatrolled users",
+        "grouppage-autopatrolled": "m:Special:MyLanguage/User groups#Autopatrolled",
 	"group-autopatrolled-member": "{{GENDER:$1|autopatrolled}}",
 	"group-confirmed": "Confirmed users",
 	"group-confirmed-member": "{{GENDER:$1|confirmed user}}",
+        "grouppage-confirmed": "m:Special:MyLanguage/User groups#(Auto)confirmed",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:119]: Tabs not spaces

File: i18n/miraheze/en.json [hunk 24]
```diff
@@ -92,52 +92,51 @@
...
 	"group-wikimanager": "Wiki Managers",
 	"group-wikimanager-member": "{{GENDER:$1|wiki manager}}",
 	"grouppage-wikimanager": "m:Wiki managers",
 	"group-rollbacker": "Rollbackers",
-        "grouppage-rollbacker": "Project:Rollbackers",
+        "grouppage-rollbacker": "m:Special:MyLanguage/User groups#Rollbackers",
 	"group-rollbacker-member": "{{GENDER:$1|rollbacker}}",
 	"group-autopatrolled": "Autopatrolled users",
-        "grouppage-autopatrolled": "Project:Autopatrolled users",
+        "grouppage-autopatrolled": "m:Special:MyLanguage/User groups#Autopatrolled",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:115]: Tabs not spaces

File: i18n/miraheze/en.json [hunk 25]
```diff
@@ -92,52 +92,51 @@
...
 	"group-wikicreator": "Wiki Creators",
 	"group-wikicreator-member": "{{GENDER:$1|wiki creator}}",
-	"grouppage-wikicreator": "m:Wiki creators",
+	"grouppage-wikicreator": "m:Special:MyLanguage/Meta:Wiki creators",
 	"group-wikimanager": "Wiki Managers",
 	"group-wikimanager-member": "{{GENDER:$1|wiki manager}}",
 	"grouppage-wikimanager": "m:Wiki managers",
 	"group-rollbacker": "Rollbackers",
-        "grouppage-rollbacker": "Project:Rollbackers",
+        "grouppage-rollbacker": "m:Special:MyLanguage/User groups#Rollbackers",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:112]: Tabs not spaces

File: i18n/miraheze/qqq.json [hunk 26]
```diff
@@ -37,38 +37,39 @@
 	"group-abuse-member": "{{GENDER:$1|abuse team}}",
```
Comment (Universal-Omega) [inline i18n/miraheze/qqq.json:37]: Nothing in this file needs Special:MyLanguage, like you added, but I suppose it could be fine.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: includes/MirahezeMagicHooks.php (hunk 5 above)
Comment (Universal-Omega) [reply includes/MirahezeMagicHooks.php]: This still needs to use single quotes, and remove the `: "<text>"` stuff.

File: i18n/overrides/qqq.json [hunk 27]
```diff
@@ -13,5 +13,21 @@
...
+	"miraheze-grouppage-bureaucrat": "m:User groups#Bureaucrats",
+	"miraheze-group-bureaucrat-member": "{{GENDER:$1|bureaucrat}}",
+	"miraheze-group-sysop": "Administrators",
+	"miraheze-grouppage-sysop": "m:User groups#Administrators",
+	"miraheze-group-sysop-member": "{{GENDER:$1|administrator}}",
+	"miraheze-group-interface-admin": "Interface administrators",
+	"miraheze-grouppage-interface-admin": "m:User groups#Interface administrators",
+	"miraheze-group-interface-admin-member": "{{GENDER:$1|interface administrator}}",
+	"miraheze-group-bot": "Bots",
+    	"miraheze-grouppage-bot": "m:User groups#Bots",
```
Comment (Universal-Omega) [inline i18n/overrides/qqq.json:30]: Tabs not spaces

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/overrides/qqq.json (hunk 27 above)
Comment (dmehus) [reply i18n/overrides/qqq.json:30]: @Universal-Omega Uhh, which file? https://github.com/dmehus/MirahezeMagic/edit/patch-1/i18n/overrides/qqq.json is done already :)

Comment (Universal-Omega) [reply i18n/overrides/qqq.json:30]: Line 30 isn't done.

Comment (dmehus) [reply i18n/overrides/qqq.json]: LOL you are observant. How did you notice that? :P

**Review by Universal-Omega — APPROVED** [review]

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
Use the anchors set now rather than `{{int:}}` anchors. As Agent has added `{{anchor}}` links to the page, which can be seen at https://meta.miraheze.org/wiki/User_groups?action=raw.

**Comment by dmehus** [comment]
> Use the anchors set now rather than `{{int:}}` anchors. As Agent has added `{{anchor}}` links to the page, which can be seen at https://meta.miraheze.org/wiki/User_groups?action=raw.

Agent is reverting that to use the localised messages. He's going to re-add the Special:MyLanguage links to my branch, then I'll sync my PR.

**Comment by Universal-Omega** [comment]
> > Use the anchors set now rather than `{{int:}}` anchors. As Agent has added `{{anchor}}` links to the page, which can be seen at https://meta.miraheze.org/wiki/User_groups?action=raw.
> 
> Agent is reverting that to use the localised messages. He's going to re-add the Special:MyLanguage links to my branch, then I'll sync my PR.

Anchor is more reliable then int due to potential deviations between the localised int on other wikis might break the anchors, using anchor tags are fixed and thus more accurate, but then again, people might still translate them if they use the anchors like that so I guess this is fine.

**Comment by dmehus** [comment]
> > > Use the anchors set now rather than `{{int:}}` anchors. As Agent has added `{{anchor}}` links to the page, which can be seen at https://meta.miraheze.org/wiki/User_groups?action=raw.
> > 
> > 
> > Agent is reverting that to use the localised messages. He's going to re-add the Special:MyLanguage links to my branch, then I'll sync my PR.
> 
> Anchor is more reliable then int due to potential deviations between the localised int on other wikis might break the anchors, using anchor tags are fixed and thus more accurate, but then again, people might still translate them if they use the anchors like that so I guess this is fine.

Yeah, I would say this is definitely more reliable, due to the potential for translation errors. Users would be linking to the localised translations anyway, so not really issue with the former. Can you resolve this?

File: i18n/overrides/en.json [hunk 28]
```diff
@@ -20,5 +20,21 @@
...
+	"miraheze-group-checkuser-member": "checkuser",
+	"miraheze-grouppage-checkuser": "m:CheckUser",
+	"miraheze-group-bureaucrat": "Bureaucrats",
+	"miraheze-grouppage-bureaucrat": "m:Special:MyLanguage/User groups#{{int:Group-bureaucrat}}",
+	"miraheze-group-bureaucrat-member": "{{GENDER:$1|bureaucrat}}",
+	"miraheze-group-sysop": "Administrators",
+	"miraheze-grouppage-sysop": "m:Special:MyLanguage/User groups#{{int:Group-sysop}}",
+	"miraheze-group-sysop-member": "{{GENDER:$1|administrator}}",
+	"miraheze-group-interface-admin": "Interface administrators",
+	"miraheze-grouppage-interface-admin": "m:Special:MyLanguage/Special:MyLanguage/User groups#{{int:Group-interface-admin}}",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:34]: double Special:MyLanguage

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/miraheze/en.json [hunk 29]
```diff
@@ -95,30 +95,28 @@
 	"grouppage-steward": "m:Stewards",
 	"group-sysadmin": "System administrators",
 	"group-sysadmin-member": "{{GENDER:$1|system administrator}}",
-	"grouppage-sysadmin": "m:System administrators",
+	"grouppage-sysadmin": "m:Tech:SRE Volunteers",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:98]: This has translations so can be Special:MyLanguage also.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/miraheze/en.json [hunk 30]
```diff
@@ -95,30 +95,28 @@
...
 	"group-interwiki-admin": "Interwiki Administrator",
 	"group-interwiki-admin-member": "interwiki administrator",
 	"grouppage-interwiki-admin": "m:Interwiki administrators",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:101]: Special:MyLanguage?

File: i18n/miraheze/en.json [hunk 31]
```diff
@@ -95,30 +95,28 @@
...
 	"group-oversight": "Oversighters",
 	"group-oversight-member": "oversight",
 	"grouppage-oversight": "m:Oversight",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:104]: Special:MyLanguage?

File: i18n/miraheze/en.json [hunk 32]
```diff
@@ -95,30 +95,28 @@
...
 	"group-autopatrolled": "Autopatrolled users",
-        "grouppage-autopatrolled": "Project:Autopatrolled users",
+	"grouppage-autopatrolled": "m:Special:MyLanguage/User groups#{{int:Group-autopatrolled}}",
 	"group-autopatrolled-member": "{{GENDER:$1|autopatrolled}}",
 	"group-confirmed": "Confirmed users",
 	"group-confirmed-member": "{{GENDER:$1|confirmed user}}",
+	"grouppage-confirmed": "m:Special:MyLanguage/User groups#{{int:Group-autoconfirmed}}",
 	"group-globalsysop": "Global Sysops",
 	"group-globalsysop-member": "{{GENDER:$1|Global Sysop}}",
 	"grouppage-globalsysop": "m:Global Sysops",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:122]: Special:MyLanguage?

File: i18n/overrides/en.json [hunk 33]
```diff
@@ -20,5 +20,21 @@
 	"miraheze-restriction-delete": "Delete",
 	"miraheze-skinname-snapwikiskin": "Snap!",
 	"miraheze-snapwikiskin": "Snap!",
-	"miraheze-wikibase-sitelinks-miraheze": "Miraheze Wikis"
+	"miraheze-wikibase-sitelinks-miraheze": "Miraheze Wikis",
+	"miraheze-group-checkuser": "Checkusers",
+	"miraheze-group-checkuser-member": "checkuser",
+	"miraheze-grouppage-checkuser": "m:CheckUser",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:26]: Special:MyLanguage?

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/miraheze/en.json (hunk 30 above)
Comment (dmehus) [reply i18n/miraheze/en.json:101]: Which file is this in? I think I changed this?

Comment (Universal-Omega) [reply i18n/miraheze/en.json:101]: > i18n/miraheze/en.json

File: i18n/miraheze/en.json (hunk 32 above)
Comment (dmehus) [reply i18n/miraheze/en.json:122]: Not sure why line 119 is highlighted, but if you mean line 122, then yeah

Comment (Universal-Omega) [reply i18n/miraheze/en.json:122]: because line 119 is what you changed, the review is for line 122

File: i18n/miraheze/en.json (hunk 30 above)
Comment (dmehus) [reply i18n/miraheze/en.json:101]: I'm so confused, some groups I'm changing in /overrides and others in /miraheze. But /miraheze is not using the localised messages and Special:MyLanguage, so what needs changing?

File: i18n/miraheze/en.json [hunk 34]
```diff
@@ -92,52 +92,51 @@
...
-	"grouppage-checkuser": "m:CheckUser",
+	"grouppage-oversight": "m:OSpecial:MyLanguage/versight",
 	"group-wikicreator": "Wiki Creators",
 	"group-wikicreator-member": "{{GENDER:$1|wiki creator}}",
-	"grouppage-wikicreator": "m:Wiki creators",
+	"grouppage-wikicreator": "m:Special:MyLanguage/Meta:Wiki creators",
 	"group-wikimanager": "Wiki Managers",
 	"group-wikimanager-member": "{{GENDER:$1|wiki manager}}",
-	"grouppage-wikimanager": "m:Wiki managers",
+	"grouppage-wikimanager": "m:WSpecial:MyLanguage/iki managers",
```
Comment (Universal-Omega) [inline i18n/miraheze/en.json:110]: Something seems wrong here...

Also, this group doesn't even exist anymore does it?

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/miraheze/en.json (hunk 34 above)
Comment (dmehus) [reply i18n/miraheze/en.json:110]: It doesn't exist, but let's leave it for now. I'll do another PR later to remove it. Nevertheless, I'll fix the mistake

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**